docs/_static/css/custom.css

```diff
@@ -1,6 +1,20 @@
 /** Fix the font weight (300 for normal, 400 for slightly bold) **/
 /** Should be 100 for all headers, 400 for normal text  **/
 
+/** We are creating both of these styles only to have the
+ *  headers of the compatibility matrix to be vertically
+ *  aligned. This should not impact the other tables
```

Comment:
I would say it slightly different:
The impact for other tables is that headers will be forcedly aligned to the bottom line, but that does not change the visualization

Or some better phrase of course

Reply:
That's a great suggestion. I have updated the PR to use it